lib/buildtools/src/build/docs/typedoc.ts

```
@@ -7,6 +7,11 @@ import * as td from 'typedoc';
 // TODO: Remove the notExported option once we decide how to handle types being exported from bundles
 // #region commonOpts
 const typedocPackageOptions: td.Configuration.TypeDocOptions = {
+  blockTags: [
+    ...td.OptionDefaults.blockTags,
+    '@publicReturnType',
+    '@publicType'
+  ],
```

Biome (2.5.3)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)